src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -0,0 +1,127 @@
+import { FC, RefObject, useState } from "react"
+import {
+  Button,
+  Flex,
+  Spacer,
+  BorderedRadio,
+  RadioGroup,
+  Text,
+  Collapse,
+  Clickable,
+} from "@artsy/palette"
+import {
+  PaymentPicker,
+  PaymentPickerFragmentContainer,
+} from "../../Components/PaymentPicker"
+import { Media } from "v2/Utils/Responsive"
+import { BankDebitProvider } from "v2/Components/BankDebitForm/BankDebitProvider"
+import { ContinueButton } from "./index"
+import { Payment_me } from "v2/__generated__/Payment_me.graphql"
+import { Payment_order } from "v2/__generated__/Payment_order.graphql"
+import { CommitMutation } from "../../Utils/commitMutation"
+
+export interface Props {
+  order: Payment_order
+  me: Payment_me
+  commitMutation: CommitMutation
+  isLoading: boolean
+  onContinue: () => void
+  paymentPicker: RefObject<PaymentPicker>
+}
+
+export const PaymentContent: FC<Props> = props => {
+  const {
+    commitMutation,
+    isLoading,
+    onContinue,
+    me,
+    order,
+    paymentPicker,
+  } = props
+  const [stepOneComplete, setStepOneComplete] = useState(false)
+  const [paymentMethodSelection, setPaymentMethodSelection] = useState("")
+  const [paymentMethod, setPaymentMethod] = useState("")
+
+  const paymentMethodLabel = (paymentMethod: string) => {
+    switch (paymentMethod) {
+      case "credit_card":
+        return "Credit Card"
+      case "bank_transfer":
```

Comment:
Ideally the engineering-facing variables and user-facing labels would match, but in reality they can diverge. I wonder if it's better to be consistent for those engineering-facing variables which we can control. For example, maybe we can use `bank_debit` as the payment method to be consistent with Exchange, and user-facing label can say "Bank Transfer" (or whatever copy desired).

Reply:
Ah okay, yeah that makes sense thank you for the suggestion!